examples/large_deformation/hyperelastic_gen_yeoh_up.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/env python
+r"""
+Incompressible generalized Yeoh hyperelastic material model [1].
+In this model, the deformation energy density per unit reference volume is
+given by
+
+.. math::
+    W =
+      K_1 \, \left( \overline I_1 - 3 \right)^{m}
+      +K_2 \, \left( \overline I_1 - 3 \right)^{p}
+      +K_3 \, \left( \overline I_1 - 3 \right)^{q}
+
+where :math:`\overline I_1` is the first main invariant of the deviatoric part
+of the right Cauchy-Green deformation tensor :math:`\ull{C}`. The coefficients
+:math:`K_1, K_2, K_3` and exponents :math:`m, p, q` are material parameters.
+Only the first term is used in this example for simplicity.
```

Comment:
Could you explain better the correspondence of the above parameters to the ones used in the term docstring (p, K == m, K_1?).

So the general model can be obtained by adding three dw_tl_he_genyeoh terms together?